hazelcast/src/main/java/com/hazelcast/internal/metrics/metricsets/RuntimeMetricSet.java

```diff
@@ -97,5 +106,10 @@ public long get(RuntimeMXBean runtimeMXBean) {
                     }
                 }
         );
+        registeredMetrics.add("runtime.uptime");
+    }
+
+    public static Set<String> getRegisteredMetricNames() {
```

Comment:
This duplication is fine because these are the keys of which the probes can be used; it is their official contract.